### DISCUSSION TITLE
2006/09/07/labview-%E7%9A%84%E8%BF%90%E8%A1%8C%E6%95%88%E7%8E%87-1-%E6%89%BE%E5%88%B0%E7%A8%8B%E5%BA%8F%E8%BF%90%E8%A1%8C%E9%80%9F%E5%BA%A6%E7%9A%84%E7%93%B6%E9%A2%88

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#112] [Announcements]
# 2006/09/07/labview-%E7%9A%84%E8%BF%90%E8%A1%8C%E6%95%88%E7%8E%87-1-%E6%89%BE%E5%88%B0%E7%A8%8B%E5%BA%8F%E8%BF%90%E8%A1%8C%E9%80%9F%E5%BA%A6%E7%9A%84%E7%93%B6%E9%A2%88

一、找到程序运行速度的瓶颈

https://blog.qizhen.xyz/2006/09/07/labview-%E7%9A%84%E8%BF%90%E8%A1%8C%E6%95%88%E7%8E%87-1-%E6%89%BE%E5%88%B0%E7%A8%8B%E5%BA%8F%E8%BF%90%E8%A1%8C%E9%80%9F%E5%BA%A6%E7%9A%84%E7%93%B6%E9%A2%88

**Comment by ruanqizhen** [comment]
之前博客下的留言：

yi
2006年09月8日 20:58 编辑
没关系啊。还是谢谢你。好文章读了，受益匪浅！

回复
Qizhen
2006年09月8日 17:09 编辑
我用 LabVIEW 编写纯软件还行，但是对于数据采集卡就没什么研究了。真是不好意思啊 ^_^*

回复
萍
2006年09月8日 11:48 编辑
蛮喜欢你的SPACE的，寓教于乐！
呵呵……

回复
yi
2006年09月8日 10:43 编辑
还想请教两个有关数据采集的问题。（前几天实验室买了一个PCI4462的采集卡，要对其编写数据采集程序。该程序要求基于双buffer采样，且使用事件中断函数对数据进行处理（分析计算+写数据库+显示等工作）。
不明白的问题：
1.双buffer采样的机制是什么？这个buffer与事先labVIEW或者LabCVI开辟的用于从计算机buffer 读取（read）数据的buffer是不同的吗？
2.event中断机制是在扫描规定数据点数之后自动触发，这于双buffer采样机制有没有影响或冲突啊？
可能还是因为我对他们的概念不清楚，请指教啊，谢谢啦！

回复
yi
2006年09月7日 23:26 编辑
还有这样的功能，开眼界了。LabVIEW学了一段时间，现在有一个CVI的项目，现在又开始搞CVI。晕了@@
